server/src/main/java/com/android/identity/wallet/server/VerifierServlet.kt

```diff
@@ -870,7 +870,7 @@ lrW+vvdmRHBgS+ss56uWyYor6W7ah9ygBwYFK4EEACI=
         )
 
         val readerX5c = singleUseReaderKeyCertChain.certificates.map { cert ->
-            Base64.from(cert.encodedCertificate.toBase64Url())
+            Base64.from(java.util.Base64.getEncoder().encodeToString(cert.encodedCertificate))
```

Comment:
We probably should use kotlin.io.encoding.Base64 for consistency.

Reply:
Yes, please don't use Java libraries.

Where is it defined how `x5c` is to be encoded?

Reply:
gotcha, swapped to kotlin.io.encoding.Base64

@davidz25 the JSON Web Signature spec says [here](https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1.6) "Each string in the array is a base64-encoded (Section 4 of [RFC4648] -- not base64url-encoded)"